migration-engine/migration-engine-tests/tests/migrations/id.rs

```diff
@@ -257,8 +259,9 @@ fn making_an_existing_id_field_autoincrement_works(api: TestApi) {
 }
 
 // Ignoring sqlite is OK, because sqlite integer primary keys are always auto-incrementing.
+// Ignoring TiDB, we can't remove auto_increment without @@tidb_allow_remove_auto_inc enabled.
 // We test this separately on cockroachdb.
-#[test_connector(exclude(Sqlite, CockroachDb))]
+#[test_connector(exclude(Sqlite, CockroachDb, TiDB))]
```

Comment:
Is there a link for this config option?
Are there alternative strategies to achieve the same outcome?

Reply:
Link of this option: https://docs.pingcap.com/tidb/dev/system-variables#tidb_allow_remove_auto_inc-new-in-v2118-and-v304

🤔 Is there any way to enable this system variable before starting testing, so we can add an additional test case for testing.

Reply:
I think for other databases this can be configured either via environment variables (in our docker-compose), or we need to create a customized Docker image that executes some SQL on startup: https://github.com/prisma/engine-images/blob/main/cockroach/22.1.0/Dockerfile#L2 + https://github.com/prisma/engine-images/blob/main/cockroach/22.1.0/prisma_init.sql But that is suuuuper unflexible and error prone - so better to avoid.

I don't think we have a global hook that can be configured via the test suite unfortunately.